var/spack/repos/builtin/packages/ascent/package.py

```diff
@@ -26,34 +34,36 @@ class Ascent(Package):
     simulations."""
 
     homepage = "https://github.com/Alpine-DAV/ascent"
-    git      = "https://github.com/Alpine-DAV/ascent.git"
```

Comment:
Undo this change.

Reply:
just to clarify, I need to add this back in and remove git from the version command?

Reply:
Yep. You don't _have_ to, but it's easier to have a single package-level git.